### PR TITLE
Add tests for the AMQP and infinispan client quickstarts

### PR DESCRIPTION
**Pull request by cescoffier**
**Checklist**:

Your pull request:

- [X] targets the `development` branch
- [X] uses the `999-SNAPSHOT` version of Quarkus
- [X] has tests (`mvn clean test`)
- [X] works in native (`mvn clean package -Pnative`)
- [X] has native tests (`mvn clean verify -Pnative`)
- [X] makes sure the documentation must not be updated
- [X] links the documentation update pull request (if needed)
- [X] updates or creates the `README.md` file (with build and run instructions)
- [-] For new quickstart, is located in the directory _component-quickstart_
- [-] For new quickstart, is added to the root `pom.xml` and `README.md`

### CODE REVIEW

File: infinispan-client-quickstart/src/test/java/org/acme/infinispanclient/InfinispanGreetingResourceTest.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+import static org.hamcrest.Matchers.is;
+
+@Testcontainers
+@QuarkusTest
+class InfinispanGreetingResourceTest {
+
+    @Container
+    public static final GenericContainer INFINISPAN = new FixedHostPortGenericContainer(
+            "jboss/infinispan-server:10.0.0.Beta3")
```
Comment (gsmet) [inline infinispan-client-quickstart/src/test/java/org/acme/infinispanclient/InfinispanGreetingResourceTest.java]: Should it use the Final now? We have moved to the Final.

Comment (cescoffier) [reply infinispan-client-quickstart/src/test/java/org/acme/infinispanclient/InfinispanGreetingResourceTest.java]: Need to check, before it was not working with final nor with any other version than this one.

Comment (cescoffier) [reply infinispan-client-quickstart/src/test/java/org/acme/infinispanclient/InfinispanGreetingResourceTest.java]: @gsmet Does not work with 1.0.0-Final, 1.0.0-Final-1, 1.0.1-Final-1

**Comment by cescoffier** [comment]
Do not merge, still facing infinispan issues.

**Comment by cescoffier** [comment]
Closing - out of date (heavily)

**Pull request closed without merging**